build/task-build-browser.js

```diff
@@ -35,6 +41,20 @@ async function buildBrowser() {
   return await webpackBuild(development ? browserDevConfig : browserProdConfig);
 }
 
+async function buildBrowserAlt() {
+  const { SRC_DIR, SHARED_DIR } = BrowserAltDirs;
+  const id = 'browser-alt';
+
+  if (!(await shouldRebuild(id, [SRC_DIR, id], [SHARED_DIR, 'ui/shared']))) {
+    logger.info(colors.green(`No changes in ${id}.`));
+    return { close: () => {} };
+  }
+
+  logger.info(colors.cyan(`Building ${id}...`));
+  const { development } = getBuildConfig();
+  return await webpackBuild(development ? browserAltDevConfig : browserAltProdConfig);
+}
```

Comment:
Yeah, although until we have a clear idea of how many frontends we want to support I'm OK with having some copy/paste in the build stuff (and it's actually surprisingly little copy/paste so kudos on that)